### ISSUE TITLE
Room dependency tree

### ISSUE THREAD

**Issue by eugeneloza** [OpenDungeons/OpenDungeons#747]
Maybe it's a good idea to make some rooms dependent on others. E.g. you can build library only after dormitory and hatchery.

**Comment by hwoarangmy** [comment]
There is already this concept with research. But I don't think we should be too strict on build order for basic buildings or the game would feel too linear.

**Comment by Bertram25** [comment]
I agree with @hwoarangmy here. Building the library first is part of the strategy, for instance.

**Comment by eugeneloza** [comment]
Ok. Just a raw idea. So closing then.

**Comment by akien-mga** [comment]
Poor @eugeneloza, I'm going through the 0.6.0 issues and I see many of your reports have been closed as invalid... Just wanted to say: keep them coming! We always need fresh input on the game, and if some of you proposals are considered duplicates of existing concepts or invalid issues, it most likely just means that we still have to make the gameplay concepts clearer :)
